app/browser/api/ledger.js

```diff
@@ -1717,7 +1717,7 @@ const onWalletProperties = (state, body) => {
 
   // unconfirmed amount
   const unconfirmed = parseFloat(body.get('unconfirmed'))
-  if (unconfirmed >= 0) {
+  if (unconfirmed > 0) {
```

Comment:
I think this change is not correct, because you could have wallet that have unconfirmed balance, let's say 10$. Then you import wallet from the backup, which have unconfirmed balance 0. New value will not be saved.

This `return` in the if block is more or less from the old code. What I would do is to change it to

```
const unconfirmed = parseFloat(body.get('unconfirmed'))
if (unconfirmed >= 0) {
    state = ledgerState.setInfoProp(state, 'unconfirmed', unconfirmed)
    if (clientOptions.verboseP) {
      console.log('\ngetBalance refreshes ledger info: ' + ledgerState.getInfoProp(state, 'unconfirmed'))
    }
}
```

Reply:
want to just PR an alt fix @NejcZdovc ?

Reply:
will push to this PR